### PR TITLE
Feature/df 2262 partial update ch fiche reactive version

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/shared/competence-items-collapse-panel/competence-items-collapse-panel.component.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export class CompetenceItemsCollapsePanelComponent implements OnInit {
 
   toggle() {
     this.isCollapsed = !this.isCollapsed;
+    console.log('emitting', this.isCollapsed);
```
Comment (fabiangotzen) [inline alv-portal-ui/src/app/competence-catalog/shared/competence-items-collapse-panel/competence-items-collapse-panel.component.ts]: Should be removed

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.ts [hunk 2]
```diff
@@ -122,8 +147,7 @@ export class ChFicheComponent extends CompetenceCatalogEditorAwareComponent impl
   };
   CompetenceType = CompetenceType;
   workEnvironmentTypes = Object.values(WorkEnvironmentType);
-  occupations$: Observable<ResolvedOccupation[]>;
-  competences$: { [index: string]: Observable<CompetenceSetWithResolvedKnowHow[]> } = defaultCompetences();
+  // competences$: { [index: string]: Observable<CompetenceSetWithResolvedKnowHow[]> } = defaultCompetences();
```
Comment (fabiangotzen) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.ts]: Should be removed?

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.ts [hunk 3]
```diff
@@ -518,56 +590,19 @@ export class ChFicheComponent extends CompetenceCatalogEditorAwareComponent impl
...
-
-  private resolveSoftskillsAndExpandPanel(messageKey: string) {
-    this.softskills$ = this.loadSoftskills().pipe(
-      tap((softskills) => {
-        this.notificationsService.success(messageKey);
-        this.collapsed.SOFTSKILLS = !softskills.length;
-      })
-    );
+  private loadBasicCompetences(competences: Competence[]): Observable<CompetenceSetWithResolvedKnowHow[]> {
+    return this.loadCompetences(CompetenceType.BASIC, competences); //todo replace with partial
```
Comment (fabiangotzen) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.ts]: If the todo is meant to stay, i'd suggest creating a technical debt task in Jira for it and writing the issue number here.

Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.ts]: thanks, good idea!

File: alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item-projectable/interactive-list-item-projectable.component.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    <span class="title-container d-flex flex-column align-items-start">
+      <small class="super-title">
+        {{ type }}
+        <ng-content select="[itemSuperTitle]"></ng-content>
+        <ng-container *ngIf="observables.isCompetenceCatalogEditor && showStatuses">
+            -
+            <span>{{isPublished | pureIf:
+              'portal.competence-catalog.publication-statuses.published':
+              'portal.competence-catalog.publication-statuses.not-published' | translate}}</span>
+            &nbsp;&&nbsp;
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item-projectable/interactive-list-item-projectable.component.html]: one `& ` too many?

Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item-projectable/interactive-list-item-projectable.component.html]: no, one is `&` and another is `&nbsp;`

Comment (cjpsycos) [reply alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item-projectable/interactive-list-item-projectable.component.html]: then use `&amp;`

**Pull request merged**